sources/packages/web/src/types/contracts/DataTableContract.ts

```diff
@@ -351,3 +351,34 @@ export const CASSupplierInformationHeaders = [
     key: "siteProtected",
   },
 ];
+
+/**
+ * Application Restriction Management header.
+ */
+export const ApplicationRestrictionManagementHeaders = [
+  {
+    title: "Restriction Type",
+    sortable: false,
```

Comment:
Just to confirm there is no sort request on any these columns?

Reply:
There is no sort requirement for this PR.